### ISSUE TITLE
SysDig

### ISSUE THREAD

**Issue by DerekRoberts** [bcgov/greenfield-pipeline#116]
https://developer.gov.bc.ca/BC-Government-Sysdig-Monitoring-Service-Definition

https://developer.gov.bc.ca/OpenShift-User-Guide-to-Creating-and-Using-a-Sysdig-Team-for-Monitoring

**Comment by DerekRoberts** [comment]
https://docs.developer.gov.bc.ca/request-quota-increase-for-openshift-project-set/#set-up-resource-monitoring-with-sysdig-monitor

**Comment by DerekRoberts** [comment]
@mishraomp You dug into sysdig, right?  Can we close this?

**Comment by DerekRoberts** [comment]
https://docs.developer.gov.bc.ca/sysdig-monitor-onboarding/

**Comment by mishraomp** [comment]
yeah I did look into it, some sample to add team members are here https://github.com/bcgov/pubcode/blob/main/sysdig/sysdig.yml
but the rest was done in the GUI of sysdig, like setting alerts
